### DISCUSSION TITLE
Power setting definition?

### DISCUSSION THREAD

**Discussion by TechMindsYT** [lora-aprs/LoRa_APRS_iGate#128] [Q&A] [answered]
Hi, I see there is a power setting (default = 20) on both the iGate and Tracker projects, what does this value effect? What are the limits?

Thanks
Matt
M0DQW

**Comment by jellekaufmann** [comment, marked as answer]
Hi!

It's the output power in dBm. The maximum output of the SX1278 is 20dBm.

The output power is set by LoRaClass::setTxPower in LoRa.cpp
